src/cli/commands/force/lightning/local/setup.ts

```diff
@@ -50,40 +48,13 @@ export class Setup extends SfdxCommand {
         `sfdx force:lightning:local:setup -p Android`
     ];
 
-    public skipBaseRequirements = false;
-    public skipAdditionalRequirements = false;
-
-    private setupSteps: BaseSetup | undefined;
-
-    public async run(direct: boolean = false): Promise<any> {
-        if (direct) {
-            await this.init(); // ensure init first
-        }
+    public async run(): Promise<any> {
```

Comment:
Since you're changing the method signature for the `run()` method, you'll need to do that for the other commands in other repos too.

I'm assuming you were going to update all other repos too since we don't want our commands to extend from `Setup` anymore (which they currently are).